.buildkite/pipeline.yml

```diff
@@ -4,7 +4,7 @@
 
 env:
   NIX_PATH: nixpkgs=https://github.com/serokell/nixpkgs/archive/master.tar.gz
```

Comment:
Oh, remind me please, why do we need it when we have `nixpkgs` in `sources.json`? I guess I had this question some time ago already :pled: Maybe it's better to add a comment here, otherwise I will ask again soon :pled:

Reply:
(I know it's unrelated to this PR :pled:)

Reply:
hmm, I don't remember now, it was required for something earlier :thinking:

Reply:
Apparently, it's unused now since we have `pkgs` in `default.nix`

Reply:
>  I guess I had this question some time ago already

Probably in another repository :yao:

Reply:
aha, I know, its used in `tests/tezos-binaries.nix`